src/element/mod.rs

```diff
@@ -436,21 +436,7 @@ impl Element {
 
     pub fn as_sequence(&self) -> Option<&Sequence> {
         match &self.value {
-            Value::SExp(SExp(s)) | Value::List(List(s)) => Some(s),
-            _ => None,
-        }
-    }
-
-    pub fn as_sexp(&self) -> Option<&SExp> {
-        match &self.value {
-            Value::SExp(sexp) => Some(sexp),
-            _ => None,
-        }
-    }
-
-    pub fn as_list(&self) -> Option<&List> {
```

Comment:
🗺️ Now that a `Value` doesn't contain a `List` or `SExp`, we can only lend out a `&Sequence`.

We could retain these methods and have them continue to return `&Sequence` when the type matches. Open to thoughts there.

Reply:
I think we should leave them out in favor of narrowing conversions. Let's not add too many ways to do the same thing unless it's absolutely necessary to do so.